sqlmesh/core/dialect.py

```diff
@@ -446,7 +447,7 @@ def parse(sql: str, default_dialect: t.Optional[str] = None) -> t.List[exp.Expre
         default_dialect: The dialect to use if the model does not specify one.
 
     Returns:
-        A list of the expressions, [Model, *Statements, Query | Jinja]
+        A list of the parsed expressions: [Model, *Statements, Query, *Statements]
```

Comment:
@izeigerman I thought this must be currently true, after the recent refactor. Can you take a quick look?